docs/data/xrootd/xrootd-authorization.md

```diff
@@ -47,73 +196,97 @@ u = <STORAGE PATH>/home/@=/ a
 # private dirs for users willing to store their data in the facility
 u xrootd <STORAGE PATH> a
 
-# This means that users in group 'biology' can do anything under the 'genomics' directory
-g biology <STORAGE PATH>/genomics a
+# This means that OSPool clients presenting a VOMS proxy can do anything under the 'osg' directory
+g /osg <STORAGE PATH>/osg a
 ```
 
 Replacing `<STORAGE PATH>` with the path to the directory that will contain data served by XRootD, e.g. `/data/xrootdfs`.
-This path is relative to the `oss.localroot` or `all.localroot` configuration values, if either one is defined in the
-xrootd config file.
+This path is relative to the [`rootdir`](install-standalone.md#configuring-xrootd).
 
-!!! note
-    Specific paths need to be specified _before_ generic paths; e.g., this does not work:
+!!! warning "Configure most to least specific paths"
+    Specific paths need to be specified _before_ generic paths.
+    For example, this line will allow all users to read the contents `/data/xrootdfs/private`:
 
             u * rl /data/xrootdfs -rl /data/xrootdfs/private
```

Comment:
This seems wrong, AFAIK the permissions go always after the path i.e.

u * /data/xrootdfs rl /data/xrootdfs/private -rl